package.json

```
@@ -6,7 +6,7 @@
   "scripts": {
     "hotReloadDev": "npx nodemon ./app.ts",
     "start": "npx tsc && cp -R views dist && node dist/app.js",
```

Splitting up this task should let us include the build step in the pipeline

```suggestion
    "build": "npx tsc && cp -R views dist",
    "start": "npm run build && node dist/app.js",
```